app/models/public_body.rb

```diff
@@ -129,6 +129,17 @@ class Translation
   #
   # [1] http://git.io/vIetK
   class Version
+    before_save :copy_translated_attributes
```

Comment:
- Commit message should explain _why_ this change is being made; not just _what_ it is doing. I assume now that `PublicBody` doesn't have e.g. a `name` column, it doesn't get copied across. It should still have a `name` method that returns the translation though, so not sure why this behaviour has changed?
- Should have specs.